mage_ai/data_preparation/templates/transformers/default.jinja

```diff
@@ -1,27 +1,25 @@
 {% extends "testable.jinja" %}
 {% block imports %}
-from pandas import DataFrame
-
 if 'transformer' not in globals():
     from mage_ai.data_preparation.decorators import transformer
 {% endblock %}
 
 {% block content %}
 @transformer
-def transform_df(df: DataFrame, *args, **kwargs) -> DataFrame:
+def transform(*args, **kwargs):
     """
     Template code for a transformer block.
 
     Add more parameters to this function if this block has multiple parent blocks.
     There should be one parameter for each output variable from each parent block.
 
     Args:
-        df (DataFrame): Data frame from parent block.
+        df (optional): Data frame from parent block.
 
     Returns:
-        DataFrame: Transformed data frame
+        Anything (e.g. df)
```

Comment:
nit: `e.g. data frame, dictionary, array, int, str, etc.`

Reply:
Will update in all the places where it says "Returns anything"